gems/pending/VMwareWebService/MiqPbmInventory.rb

```diff
@@ -0,0 +1,32 @@
+require 'VMwareWebService/PbmService'
+
+module MiqPbmInventory
+  def pbm_initialize(server, username, password)
+    @pbm_svc = PbmService.new(server, username, password) if @apiVersion >= '5.5'
```

Comment:
So, `RbVmomi::VIM` returns another VIM connection, and we're calling it just to get a session cookie? Then, once we can extract the existing cookie from the current VIM session we won't need to open another connection?